pkg/collector/py/tagger.c

```diff
@@ -27,7 +50,8 @@ static PyObject *get_tags(PyObject *self, PyObject *args) {
 }
 
 static PyMethodDef taggerMethods[] = {
-  {"get_tags", get_tags, METH_VARARGS, "Get tags for an entity."},
+  {"tag", tag, METH_VARARGS, "Get tags for an entity."},
```

Comment:
+1 for using the same semantic we have in Goland where you'd call `tagger.Tag()`.